src/main/java/org/truffleruby/core/symbol/SymbolTable.java

```diff
@@ -147,4 +149,11 @@ public static String checkClassVariableName(
         return name;
     }
 
+    @TruffleBoundary(transferToInterpreterOnException = false)
```

Comment:
Why transferToInterpreterOnException `false` here?

Reply:
Had it the wrong way around in my head. Fixed.